lessons/fundamentals/lesson-98-frr/lesson.meta.yaml

```
@@ -0,0 +1,50 @@
+---
+lessonName: FR in NRELabs
```

I know this is a PoC, but maybe rename to the below? Also fixing the `FR` typo.

```suggestion
lessonName: Introduction to Free-Range Routing (FRR)
```